### ISSUE TITLE
Dask .to_parquet method does not recognize kwargs from fastparquet write function

### ISSUE THREAD

**Issue by Matthew-gunsuSon** [dask/fastparquet#451]
While I was writing a parquet file from dask dataframe, I got an OSError that suggests to reduce row_group_offsets. 

I thought I could use `ddf.to_parquet('./path/myparquetfile.parquet', row_group_offsets = 1000000)`, but it does not recognize this keyword argument, saying, 
`TypeError: make_metadata() got an unexpected keyword argument 'row_group_offsets'`

So I used write function from fastparquet and it seems to work. 
`from fastparquet import write`
`write('./path/myparquetfile.parquet', ddf, row_group_offsets=1000000)`

I thought it would be convenient to use keyword argument directly from dask method. 
Is there a reason that this kwarg is not recognized on dask?

**Comment by martindurant** [comment]
> Is there a reason that this kwarg is not recognized on dask

Yes - Dask partitions data-frames anyway, into pieces that then become the row-groups. You could repartition. It is conceivable, but less likely, that users would want multiple row groups per Dask partition - but this could be enabled.

**Comment by Matthew-gunsuSon** [comment]
> > Is there a reason that this kwarg is not recognized on dask
> 
> Yes - Dask partitions data-frames anyway, into pieces that then become the row-groups. You could repartition. It is conceivable, but less likely, that users would want multiple row groups per Dask partition - but this could be enabled.

Oh, I didn't reach to think about repartitions. IMHO, it would be helpful to suggest increasing dataframe repartition instead of adjusting row_group_offsets in the error message?

**Comment by martindurant** [comment]
You certainly have a point there, although it would be a little tricky to achieve, since the error is within fastparquet, but the repartition would be in Dask. A generic message like "number of rows too large" may do, or perhaps cause even more confusion...

In fact, the right thing would be to split each row group into multiple pages (an inner division of the data) within the writer, but I don't intend to implement that.
